application/tests/api/ResetCest.php

```diff
@@ -115,77 +127,79 @@ public function test7(ApiTester $I)
     public function test72(ApiTester $I)
     {
         $I->wantTo('check response when making authenticated PUT request to resend the verification');
-        $I->haveHttpHeader('Authorization', 'Bearer user1');
+        $I->setCookie('access_token', 'user1', [
+          'expire' => time() + 3600,  // Cookie expires in 1 hour
+          'httpOnly' => true          // Cookie is not accessible via JavaScript
+        ]);
```

Comment:
if it's not difficult, this would be good to extract to a separate function

Reply:
not sure why setting a var in _bootstrap.php isn't working, but apparently it's too hard for me at the moment. I was just planning on reusing the `[
          'expire' => time() + 3600,  // Cookie expires in 1 hour
          'httpOnly' => true          // Cookie is not accessible via JavaScript
        ]` part as `$cookieConfig`